runtime/near-vm-logic/src/logic.rs

```diff
@@ -2491,7 +2491,7 @@ impl<'a> VMLogic<'a> {
 
         near_o11y::io_trace!(
             storage_op = "read",
-            key = %near_primitives::serialize::to_base58(&key),
+            key = %near_o11y::pretty::Bytes(&key),
```

Comment:
yeah I think the cases where base58 would be useful are _very_ rare

but here ASCII display could actually be really useful

Reply:
I can see arguments for either really.  There may even be cases where base58 could make sense.  Perhaps someone uses block hashes as keys?  I’m happy to just go with base64.

Reply:
yeah base58 is possible, but almost certainly it will be in a collection that is prefixes keys with something anyway

ascii keys are very common, though, as the collection prefix is usually also ascii and my current workflow of looking at those traces often involves (1) decode base 58 (2) convert bytes to ascii, which typically reveals good insight into what the contract is doing

Reply:
OK, how about now?

Reply:
yeah the separation between `Bytes` and `StorageKey` make sense to me!